MusicApp.PlaylistService.Infrastructure/Repositories/PlaylistRepository.cs

```diff
@@ -1,17 +1,21 @@
 ﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Caching.Distributed;
 using MusicApp.PlaylistService.Application.Repositories;
 using MusicApp.PlaylistService.Domain.Entities;
 using MusicApp.PlaylistService.Infrastructure.Data;
+using MusicApp.PlaylistService.Infrastructure.Extensions;
 
 namespace MusicApp.PlaylistService.Infrastructure.Repositories;
 
 public class PlaylistRepository : BaseRepository<Playlist>,  IPlaylistRepository
 {
     private readonly AppDbContext _appContext;
+    private readonly IDistributedCache _cache;
```

Comment:
mark _appContext and _cache in BaseRepository as protected and use them instead of defining these fields in each derived repository, sorry didn't notice it earlier